sherpa/fit.py

```diff
@@ -417,15 +422,16 @@ def myformat( hfmt, str, lowstr, lownum, highstr, highnum ):
         elif False == in_low and True == in_high and mymethod:
```

Comment:
How about changing the boolean checks - e.g. here

    elif not in_low and in_high and mymethod:

there's a few more like this in this block of code.